### PR TITLE
IT'S ALIVE!

### PR DESCRIPTION
**Pull request by J35P312**
Godagens!
I tested running the RNA pipeline on Rasta. 
Star adds some stuff to the prefix of the output bam, so I needed to add those things to the input of Picard tools.
But now it works perfect!

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Star_aln.pm [hunk 1]
```diff
@@ -328,7 +328,7 @@ sub analysis_star_aln {
         picardtools_addorreplacereadgroups(
             {
                 FILEHANDLE  => $FILEHANDLE,
-                infile_path => $outfile_path_prefix . $outfile_suffix,
+                infile_path => $outfile_path_prefix .q{.Aligned.sortedByCoord.out}  . $outfile_suffix,
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Star_aln.pm]: .q{.Aligned.sortedByCoord.out}  . -> . $DOT . q{Aligned.sortedByCoord.out} .

**Comment by henrikstranneheim** [comment]
fabulous!!

**Comment by J35P312** [comment]
Thanks! Now I have changed the dot into a $DOT!

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**